docs/sources/reference/playlist.md

```diff
@@ -11,59 +11,96 @@ weight = 4
+1. Click __Add to playlist__ next to the dashboard you want to add.
+   - Click on __Remove[x]__ to remove a dashboard from the playlist.
 
-Since the Playlist is basically a list of Dashboards, ensure that all the Dashboards you want to appear in your Playlist are added here.
+The Playlist is essentially a list of dashboards. Be sure that all the dashboards you want to appear in your playlist are added in this section before saving the playlist.
 
 ## Save a playlist
 
-Once all the wanted dashboards are added to a playlist, you can save this playlist by clicking on the green **Save** button. This will generate a unique URL for you playlist which can be shared if needed. Click on the generated URL or on the **Play** button from the **Saved playlists** list to start the playlist. If you want to share the URL, right click on the URL and copy the URL link and share.
+To save a playlist: 
+
+1. Ensure that your playlist has a __Name__, __Interval__, and at least one __Dashboard__ added to it.
+1. Click the green __Create__ button.
+
+   This will generate a unique, shareable URL for your playlist.
+
+ Click on the generated URL to start the Playlist from here. If you want to share the URL, right click on the URL and copy the URL link and share.
 
 ## Start a playlist
 
-Also, if you want, you can start the playlist without saving it by clicking on the green **Start** button at the bottom.
+To start a playlist: 
 
-## Control a playlist
+1. From the Dashboards submenu, click __Playlists__.
+1. Next to the playlist you want to start, click __Start playlist__.
+1. In the dropdown, select the mode you want the playlist to display in.
+   - In Normal mode
+   - In TV mode
+   - In TV mode (with auto fit panels)
+   - In Kiosk mode
+   - In Kiosk mode (with auto fit panels)
+
+### Normal, TV and Kiosk mode
+
+__Normal mode:__ The side menu remains visible. The navbar, row and panel controls appear at the top of the screen.
+
+__TV mode:__ The side mmenu is hidden/removed. The navbar, row and panel controls appear at the top of the screen.
 
-Playlists can also be manually controlled utilizing the Playlist controls at the top of screen when in Playlist mode.
+A playlist enters __TV mode__ automatically after one minute of user inactivity. To toggle it manually, 
+use the `d v` sequence shortcut, or append the parameter `?inactive` to the dashboard URL. Restore the navbar and controls with any mouse movement or keyboard action.
 
-Click the stop button to stop the Playlist, and exit to the current Dashboard.
-Click the next button to advance to the next Dashboard in the Playlist.
-Click the back button to rewind to the previous Dashboard in the Playlist.
+__Kiosk mode:__ The side menu, navbar, row and panel controls are completely hidden/removed from view. 
+
+To put a playlist into __Kiosk__ mode, use the `d k` shortcut after the playlist has started. Toggle the playlist out of kiosk mode with the same shortcut.
+
+## Control a playlist
```

Comment:
Playlist controls might be a good reference topic, or add it to the concept topic. They are not really appropriate for numbered steps. 

Numbered lists are used when the order of the steps is important.
Unordered lists are used when the steps can be completed in any order.

This particular information might best be served with either a table showing the buttons and what that they do or with a screenshot and orange numbers as I have seen in other parts of the documentation.